modules/neo4j/src/main/java/org/testcontainers/containers/Neo4jContainer.java

```diff
@@ -0,0 +1,174 @@
+
+    /**
+     * The port of the transactional HTTP endpoint: <a href="https://neo4j.com/docs/rest-docs/current/">Neo4j REST API</a>.
+     */
+    private static final int DEFAULT_HTTP_PORT = 7474;
+
+    /**
+     * The official image requires a change of password by default from "neo4j" to something else. This defaults to "password".
+     */
+    private static final String DEFAULT_ADMIN_PASSWORD = "password";
+
+    private static final String AUTH_FORMAT = "neo4j/%s";
+
+    private String adminPassword = DEFAULT_ADMIN_PASSWORD;
+
+    private boolean defaultImage = false;
+
+    /**
+     * Creates a testcontainer using the official Neo4j docker image.
+     */
+    public Neo4jContainer() {
+        this(DOCKER_IMAGE_NAME);
+
+        this.defaultImage = true;
+    }
+
+    /**
+     * Creates a testcontainer using a specific docker image.
+     *
+     * @param dockerImageName The docker image to use.
+     */
+    public Neo4jContainer(String dockerImageName) {
+        super(dockerImageName);
+
+        WaitStrategy waitForBolt = new LogMessageWaitStrategy()
+            .withRegEx(String.format(".*Bolt enabled on 0\\.0\\.0\\.0:%d\\.\n", DEFAULT_BOLT_PORT));
+        WaitStrategy waitForHttp = new HttpWaitStrategy()
+            .forPort(DEFAULT_HTTP_PORT)
+            .forStatusCodeMatching(response -> response == HTTP_OK);
+
+        this.waitStrategy = new WaitAllStrategy()
+            .withStrategy(waitForBolt)
+            .withStrategy(waitForHttp)
+            .withStartupTimeout(Duration.ofMinutes(2));
+    }
+
+    @Override
+    public Set<Integer> getLivenessCheckPortNumbers() {
+
+        return Stream.of(DEFAULT_BOLT_PORT, DEFAULT_HTTP_PORT, DEFAULT_HTTPS_PORT)
+            .map(this::getMappedPort)
+            .collect(toSet());
+    }
+
+    @Override
+    protected void configure() {
+
+        addExposedPorts(DEFAULT_BOLT_PORT, DEFAULT_HTTP_PORT, DEFAULT_HTTPS_PORT);
+
+        boolean emptyAdminPassword = this.adminPassword == null || this.adminPassword.isEmpty();
+        String neo4jAuth = emptyAdminPassword ? "none" : String.format(AUTH_FORMAT, this.adminPassword);
+        addEnv("NEO4J_AUTH", neo4jAuth);
+    }
+
+    /**
+     * @return Bolt URL for use with Neo4j's Java-Driver.
+     */
+    public String getBoltUrl() {
+        return String.format("bolt://" + getContainerIpAddress() + ":" + getMappedPort(DEFAULT_BOLT_PORT));
+    }
+
+    /**
+     * @return URL of the transactional HTTP endpoint.
+     */
+    public String getHttpUrl() {
+        return String.format("http://" + getContainerIpAddress() + ":" + getMappedPort(DEFAULT_HTTP_PORT));
+    }
+
+    /**
+     * @return URL of the transactional HTTPS endpoint.
+     */
+    public String getHttpsUrl() {
+        return String.format("https://" + getContainerIpAddress() + ":" + getMappedPort(DEFAULT_HTTPS_PORT));
+    }
+
+    /**
+     * Configures the container to use the enterprise edition of the default docker image.<br />
```

Comment:
Self-closing tags break the Javadoc generation of the build:
```
/home/travis/build/testcontainers/testcontainers-java/modules/neo4j/src/main/java/org/testcontainers/containers/Neo4jContainer.java:129: error: self-closing element not allowed
     * Configures the container to use the enterprise edition of the default docker image.<br />
```

Reply:
Some of the things I'll get wrong every second time.